libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -230,7 +230,7 @@ void AP_MotorsHeli_Dual::set_update_rate( uint16_t speed_hz )
 }
 
 // init_outputs
-bool AP_MotorsHeli_Dual::init_outputs()
```

Comment:
The "AP_MotorsHeli_Dual::init_outputs()" one is a little different than the rest in that it does not have the early return.  So this is the one that we need to check closely to be sure that it is OK for the stuff below to run.  So for example this is the only one that allows the initialised_ok to move back to being false if the user change the frame parameter (I think).

Reply:
Yeah, we should probably fix up the others to allow them to move back to false too. We should also probably not be continually adding motors and initting servos in the case that it is false.

I didn't put a large amount of thought into this bit except removing the returns as there all the same.

More tidy ups coming in the future.